webapp/src/containers/Rhs/projectDetails/header.tsx

```diff
@@ -96,9 +137,12 @@ const Header = ({projectDetails, showAllSubscriptions, handlePagination, setShow
                 onConfirm={handleConfirmUnlinkProject}
                 isLoading={isUnlinkProjectLoading}
                 confirmBtnText='Unlink'
-                description={`Are you sure you want to unlink ${projectName}?`}
+                description={confirmationModalDescription}
                 title='Confirm Project Unlink'
-            />
+                showErrorPanel={unlinkConfirmationModalError}
+            >
+                {showDeleteSubscriptionsCheckbox ? deleteSubscriptionsCheckbox : <></>}
```

Comment:
```suggestion
                {showDeleteSubscriptionsCheckbox && deleteSubscriptionsCheckbox}
```

Reply:
check if this works

Reply:
It gives type error that children cannot be boolean